sebs/cloudflare/r2.py

```
@@ -0,0 +1,407 @@
+                self.logging.error(f"Failed to create R2 bucket: {bucket_info_json.get('errors')}")
+                raise RuntimeError(f"Failed to create R2 bucket {name}")
+
+            bucket_name = bucket_info_json.get("result", {}).get("name", name)
+            self.logging.info(f"Created R2 bucket {bucket_name}")
+            return bucket_name
+            
+        except requests.exceptions.RequestException as e:
+            self.logging.error(f"Error creating R2 bucket {name}: {e}")
+            raise
+
+    def download(self, bucket_name: str, key: str, filepath: str) -> None:
```

Ruff (0.14.13)

Unused method argument: `key`

(ARG002)